ani-cli

```diff
@@ -63,12 +63,19 @@ help_info() {
         play dubbed version
       -U, --update
         Update the script
+      -e --season
```

Comment:
`-e` is already take for `--episode`.
I'm also not sure season selection work well if at all, since many anime don't have numbered seasons, but rather differently named continuations.
If we can have `-o` without `-e`, that would be prefered.
Also consider `-f` for `format` over `-o` for `output`, tho I have no strong preference on this

Reply:
Sorry didnt notice that -e it was taken. And i dont really have a preference either for the -f or -o thing. If people want it i guess i could change it
I will change the -e but not sure what i can change it with